RetailCoder.VBE/Navigation/CodeMetrics/CodeMetricsResult.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Rubberduck.Parsing.Symbols;
+using Rubberduck.VBEditor;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Rubberduck.Navigation.CodeMetrics
+{
+    public struct CodeMetricsResult
+    {
+        public CodeMetricsResult(int lines, int cyclomaticComplexity, int nesting)
```

Comment:
Why are we hard-coding the types of code metrics we support?

Reply:
because aggregation may follow different rules depending on the metric. Since that's the case, we'd need to accomodate an aggregation in all kinds of metrics. 

Last but not least this seemed like the least effort. This PR is "only" a first iteration of the feature, and future iterations may well change this. But I don't think we can reasonably expect user-contributed code-metrics. It's just not something I think we're going to need.